### ISSUE TITLE
Strange warning message: line -1 with orphan callout

### ISSUE THREAD

**Issue by jmini** [asciidoctor/asciidoctor#2275]
Minimal example to reproduce it. You need 3 Files:

* `example-manual.adoc`
* `_file1.adoc`
* `_file2.adoc`

Content for `example-manual.adoc`:
```
= Example Manual
Doc Writer <doc.writer@example.org>
2014-09-09
:revnumber: {project-version}

== Introduction

include::_file1.adoc[]

include::_file2.adoc[]
```

Content for `_file1.adoc`:
```
Lorem

// missing listing

<1> one
<2> two
```

Content for `_file2.adoc`:
```
Lorem ipsum
```

Error message I get:
```
asciidoctor: WARNING: _file1.adoc: line 4: no callouts refer to list item 1
asciidoctor: WARNING: _file2.adoc: line -1: no callouts refer to list item 2
```

The second line is really strange. The problematic callout `<2>` is located on line 6 of `_file1.adoc`.

**Comment by mojavelinux** [comment]
This was fixed in 1.5.6 with commit 2a7d41db0d9bf26e88d1187a3d998965ba6c1045. I just forgot to reference the issue.

**Comment by mojavelinux** [comment]
Thanks for reporting, btw!
